drivers/flash/flash_stm32_ospi.c

```diff
@@ -319,21 +320,22 @@ static int stm32_ospi_mem_ready(OSPI_HandleTypeDef *hospi, uint8_t nor_mode, uin
 	/* Configure automatic polling mode command to wait for memory ready */
 	if (nor_mode == OSPI_OPI_MODE) {
 		s_command.Instruction = SPI_NOR_OCMD_RDSR;
+		s_command.AddressMode = HAL_OSPI_ADDRESS_8_LINES;
```

Comment:
`ospi_prepare_cmd` already sets `s_command.AddressMode = HAL_OSPI_ADDRESS_8_LINES` for `OSPI_OPI_MODE`.

Reply:
ok